federationagent.json

```diff
@@ -0,0 +1,27 @@
+{
+    "extends": ["config:base", ":label(dependencies)"],
+    "rebaseWhen": "conflicted",
+    "prConcurrentLimit": 5,
+    "commitMessageTopic": "{{depName}}",
```

Comment:
Should we add the same fields on lib & policy config files?

Reply:
we can 🤷  but that's not the purpose of this PR :p

Reply:
yes I know, it was just a thought I had